cps/uploader.py

```diff
@@ -320,22 +322,36 @@ def video_metadata(tmp_file_path, original_file_name, original_file_extension):
             identifiers=[])
         return meta
 
+def sanitize_path(path):
+    """Sanitize the file path to prevent command injection."""
+    return shlex.quote(path)
+
 def generate_video_cover(tmp_file_path):
     ffmpeg_executable = os.getenv('FFMPEG_PATH', 'ffmpeg')
-    ffmpeg_output_file = os.path.splitext(tmp_file_path)[0] + '.cover.jpg'
+    if not ffmpeg_executable:
```

Comment:
Because getenv() returns "ffmpeg" as default, this check will never succeed.

Either remove this check, or change the default to "".

Reply:
Good point